v0/destinations/attentive_tag/transform.js

```diff
@@ -0,0 +1,123 @@
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG, BASE_URL } = require("./config");
+const get = require("get-value");
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  constructPayload,
+  defaultPostRequestConfig,
+  removeUndefinedAndNullValues,
+  getIntegrationsObj,
+  CustomError
+} = require("../../util");
+
+const identifyResponseBuilder = (message, { Config }) => {
+  const { apiKey, signUpSourceId } = Config;
+  let endpoint; 
+  const integrationsObj = getIntegrationsObj(message, "attentive_tag");
```

Comment:
Add canonical names also in constants/destinationCanonicalNames.js

Reply:
Added